go/database/sql/README.md

```diff
@@ -40,15 +39,24 @@ type CommenterOptions struct {
 	EnableController  bool   // applicable for web frameworks
 	EnableAction      bool   // applicable for web frameworks
 	EnableApplication bool   // applicable for web frameworks
-	Application       string // user-provided application-name. optional
-	}
+}
+```
+
+Users can also provide static tags they want to use by using `core.StaticTags` struct. These tags are optional
```

Comment:
Maybe we should sometime describe how these static tags helps. There is some logic in the calculation part regarding both Application and DBDriver, where the library will try to find these tags if not present.

Reply:
fixed